### PR TITLE
#19 Show suggested users

### PR DESCRIPTION
**Pull request by OussamaJmili**
#19 #119 

* Added suggested users section in right side
* Added 7 filters (common users, industry, school, courses, skills, languages, and experience level) that would show suggested users.
* styled the section
* commented the code

update:
* fixed the big number of reads from the db
* added the connect functionality

todo: remove a user from suggested users if connection is pending

### CODE REVIEW

File: linkedin-clone/src/components/Rightside.js [hunk 1]
```diff
@@ -1,9 +1,207 @@
...
+
+    
+
+
+    return () => {
+      unsubscribe();
+      window.removeEventListener('resize', handleResize);
+    }
+
+  }, [displayedUsers]);
```
Comment (drbladz) [inline linkedin-clone/src/components/Rightside.js]: Make sure this does not cause infinite loops. You can remove the parameter if not needed here.

Comment (OussamaJmili) [reply linkedin-clone/src/components/Rightside.js]: it's needed to display the right amount of users depending on the width of the window

Comment (drbladz) [reply linkedin-clone/src/components/Rightside.js]: Can you console log in your useEffect to see if the function repeats? Let me know

Comment (OussamaJmili) [reply linkedin-clone/src/components/Rightside.js]: problem resolved

**Review by drbladz — APPROVED** [review]
Looks good. Nice work on filtering suggested users based on common interests.

**Review by TNT05 — APPROVED** [review]
looks clean, the scoring system is very professional

**Pull request merged**